archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -0,0 +1,87 @@
+import tactic
```

Comment:
```suggestion
/-
Copyright (c) 2015 Joe Cool. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Jalex Stark,

Solving problem 42 on the list of 100 theorems, which involves a "telescoping" sum.
-/
import tactic
```

Reply:
> Solving problem 42 on the list of 100 theorems, which involves a "telescoping" sum.

This text should go in a separate module doc after the imports.